src/main/keybinds.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * SPDX-License-Identifier: GPL-3.0
+ * Vesktop, a desktop app aiming to give you a snappier Discord Experience
+ * Copyright (c) 2023 Vendicated and Vencord contributors
+ */
+
+import child_process from "node:child_process";
+import fs from "node:fs";
+import path from "node:path";
+
+import net from "net";
+import { IpcEvents } from "shared/IpcEvents";
```

Comment:
always prefer picking only the imports you need instead of importing the entire module

```suggestion
import { spawnSync } from "node:child_process";
import { open } from "node:fs";
import { join } from "node:path";

import { Socket } from "net";
import { IpcEvents } from "shared/IpcEvents";
```